app/src/main/java/com/github/sdpteam15/polyevents/database/observe/ObservableList.kt

```diff
@@ -0,0 +1,253 @@
+package com.github.sdpteam15.polyevents.database.observe
+
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.OnLifecycleEvent
+import com.github.sdpteam15.polyevents.helper.HelperFunctions.observeOnStop
+import com.github.sdpteam15.polyevents.helper.HelperFunctions.run
+import java.util.*
+
+class ObservableList<T> {
```

Comment:
Maybe add a top level class comment to explain what it's used for (e.g.  in relation of this class with the Observable)?